parachains/integration-tests/emulated/assets/asset-hub-kusama/src/tests/teleport.rs

```diff
@@ -185,6 +185,7 @@ fn limited_teleport_native_assets_from_relay_to_system_para_works() {
 /// Limited Teleport of native asset from System Parachain to Relay Chain
 /// should work when there is enough balance in Relay Chain's `CheckAccount`
 #[test]
+#[cfg(feature = "FIXME-IGNORED")] // <https://github.com/paritytech/cumulus/issues/3027>
```

Comment:
Agreed seems fine as long as runtime-benchmarks is not enabled:
```suggestion
#[cfg(not(feature = "runtime-benchmarks"))] // <https://github.com/paritytech/cumulus/issues/3027>
```

Reply:
The crate does not have that feature and is also not propagating it downwards...